test/unit/BookList.test.js

```diff
@@ -34,9 +35,8 @@ describe('BookList', () => {
       expect(component.find('.nypl-column-three-quarters').length).to.equal(1);
     });
 
-    it('should have an h2', () => {
-      expect(component.find('h2').length).to.equal(1);
-      expect(component.find('h2').text()).to.equal(' Picks for 0 Books Found');
+    it('should render an <ListTitle> component', () => {
+      expect(component.find(ListTitle).length).to.equal(1);
```

Comment:
Here you imported the component and then used it inside `.find()`. But in the previous tests, you don't import the components and use them as strings in `.find()`. I prefer importing them and checking for those components. Just be consistent.

Reply:
Can you point me where the previous tests you see? Do we need to update them in this PR?

Reply:
Yeah as I mentioned I just reused the same tests from previous dev's tests but expand them to different params. I can update them